README.md

```diff
@@ -743,6 +744,17 @@ const tail = arr => arr.length > 1 ? arr.slice(1) : arr;
 // tail([1]) -> [1]
 ```
 
+### Take
+
+Use `Array.slice()` to create a slice of the array with `n` elements taken from the beginning.
```

Comment:
As a note for the future refer to the first bullet under *Additional guidelines and conventions regarding snippets* of [CONTRIBUTING.md](https://github.com/Chalarangelo/30-seconds-of-code/blob/master/CONTRIBUTING.md#additional-guidelines-and-conventions-regarding-snippets) when referring to `Array` methods. Apart from that, solid work! Thanks for contributing yet again!